exchange/utils.py

```diff
@@ -52,9 +52,17 @@ def insert_many(objects, using="default"):
     table = model._meta.db_table
     column_names = ",".join(con.ops.quote_name(f.column) for f in fields)
     placeholders = ",".join(("%s",) * len(fields))
-    con.cursor().executemany("insert into %s (%s) values (%s)"
-                             % (table, column_names, placeholders), parameters)
-    transaction.commit_unless_managed(using=using)
+
+    from django import VERSION
+    # see https://docs.djangoproject.com/en/1.9/internals/deprecation/#deprecation-removed-in-1-8
+    if VERSION[2] < 7:
```

Comment:
Thank you for the contribution. How about:
```
from distutils.version import StrictVersion
from django import get_version

if StrictVersion(get_version()) >= StrictVersion('1.6.0'):
    with transaction.atomic():
         ....
```

https://docs.djangoproject.com/en/1.6/topics/db/transactions/#controlling-transactions-explicitly

On the other hand maybe we should not need to support < Django 1.6 at all.